frontend/fingertips-frontend/README.md

```diff
@@ -119,3 +127,35 @@ These server pages are responsible for making any calls to fetch any data. Then
 The page components are pure react components and must have the `use client` directive at the top. This is needed for the purpose of using the `govuk-react` component library. This library uses `styled-components` and react hooks and therefore need to be client components.
 
 However, Next.js will still use these component to render the page server-side.
+
+## Auto generate scripts
```

Comment:
Are there any documentation links we can provide to the tools being used here (or even just the names of them)? I think it would be good to point people at the tools being used and where they can find information on how to configure them.